src/CribblyBackend.DataAccess/Migrations/20210318091908_AddStandingsData.cs

```diff
@@ -0,0 +1,29 @@
+using FluentMigrator;
+
+namespace CribblyBackend.Migrations
+{
+    [Migration(20210318091908)]
+    public class AddStandingsData : Migration
+    {
+        public override void Down()
+        {
+            Delete.Column("Division").FromTable("Teams");
+            Delete.Column("Wins").FromTable("Teams");
+            Delete.Column("Losses").FromTable("Teams");
+            Delete.Column("TotalScore").FromTable("Teams");
+            Delete.Column("Seed").FromTable("Teams");
+            Delete.Column("InTournament").FromTable("Teams");
+        }
+
+        public override void Up()
+        {
+            Alter.Table("Teams")
+                .AddColumn("Division").AsString()
```

Comment:
Do you think it's more useful to have the division name here or the division ID that can be used to fetch the entire divisions?

Reply:
Yeah you are probably right. I will change it